autofs-5.1.3 - set systemd KillMode to process

From: Ian Kent <raven@themaw.net>

It's been reported that fuse based filesystem mounts are lost when
restarting autofs.

This behaviour is due to systemd default KillMode being control-group
rather than process.

While changing this from control-group to process might occassionally
not kill some processes that should be killed on shutdown or restart
these processes are much less common and less problematic than processes
managing fuse based mounts.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG                 |    1 +
 samples/autofs.service.in |    1 +
 2 files changed, 2 insertions(+)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -251,6 +251,7 @@
 - limit getgrgid_r() buffer size.
 - add congigure option for limiting getgrgid_r() stack usage.
 - fix unset tsd group name handling.
+- set systemd KillMode to process.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/samples/autofs.service.in
+++ autofs-5.0.7/samples/autofs.service.in
@@ -9,6 +9,7 @@ PIDFile=@@autofspiddir@@/autofs.pid
 EnvironmentFile=-@@autofsconfdir@@/autofs
 ExecStart=@@sbindir@@/automount $OPTIONS --pid-file @@autofspiddir@@/autofs.pid
 ExecReload=/usr/bin/kill -HUP $MAINPID
+KillMode=process
 TimeoutSec=180
 
 [Install]
